src/turnin.c

```diff
@@ -465,6 +473,13 @@ void setup(char *arg) {
 		        "        Please report this issue to the system administrators\n", user_uid);
 		exit(1);
 	}
+
+	if (!pwd->pw_name) {
+		fprintf(stderr, 
+					  "turnin: Cannot lookup user_name (uid %d)\n"
+						" 			 Please report this issue to the system administrators\n", user_uid);
```

Comment:
I think these are not aligned with the rest of the file.

Reply:
Yes you are right, it looked fine on my end. Is it ok if I include a `.clang-format` to avoid same issues in the future?

Reply:
@zakkak What do you think?

Reply:
+1 ideally it should be accompanied by an [`.editorconfig`](https://editorconfig.org/#example-file) file